app/views/organisers/award_badges.slim

```diff
@@ -0,0 +1,29 @@
+.page-header
+  h2.text-center.green-text Award Badges
+
+= link_to "Back to All projects", dashboard_organisers_path
+.row
+  .col-xs-30
+    .table-responsive
+      table.table.table-striped
+        thead
+          tr
+            th.col-md-1.col-md-offset-2.col-sm-1.col-sm-offset-1.col-xs-1 #
+            th.col-md-2.col-md-offset-2.col-sm-2.col-sm-offset-1.col-xs-2 Mentee
+            th.col-md-100.col-md-offset-2.col-sm-100.col-sm-offset-1.col-xs-100 Project Description
+            th.col-md-6.col-md-offset-2.col-sm-6.col-sm-offset-1.col-xs-6 
+        tbody
+        - if @projects.any?
```

Comment:
that should be nested inside `tbody` - currently it's on the same level so following `tr` tags won't be children of `tbody`